tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -5243,6 +5272,12 @@ public function test_resolve_variables() {
 
 		$this->assertEquals( $small_font, $styles['blocks']['core/quote']['variations']['plain']['typography']['fontSize'], 'Block variations: font-size' );
 		$this->assertEquals( $secondary_color, $styles['blocks']['core/quote']['variations']['plain']['color']['background'], 'Block variations: color' );
+		/*
+		 * As with wp_get_global_styles(), WP_Theme_JSON::resolve_variables may be called with merged data from
+		 * WP_Theme_JSON_Resolver. WP_Theme_JSON_Resolver::get_block_data() sets blockGap for supported blocks to `null` if the value is not defined.
+		 */
+		$this->assertNull( $styles['blocks']['core/post-template']['spacing']['blockGap'], 'core/post-template block: blockGap' );
```

Comment:
```suggestion
		$this->assertNull( $styles['blocks']['core/post-template']['spacing']['blockGap'], 'core/post-template block: blockGap', 'Post Template block's spacing blockGap should be null' );
```

Reply:
Nit: we should include the $message param.

> All PHPUnit assertions, as well as all WordPress custom assertions, allow for a $message parameter to be passed. This message will be displayed when the assertion fails and can help immensely when debugging a test. This parameter should always be used if more than one assertion is used in a test method.

https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-methods